services/port_manager/src/main/java/com/futurewei/alcor/portmanager/processor/DataPlaneProcessor.java

```diff
@@ -278,14 +313,15 @@ private void updateNetworkConfig(PortContext context, NetworkConfiguration netwo
             networkConfig.setOpType(Common.OperationType.UPDATE);
             IRestRequest updateNetworkConfigRequest =
                     new UpdateNetworkConfigRequest(context, networkConfig);
-            context.getRequestManager().sendRequestAsync(updateNetworkConfigRequest, request -> portService.updatePortStatus(request,networkConfig,null));
-            portService.updatePortStatus(updateNetworkConfigRequest,networkConfig, StatusEnum.PENDING.getStatus());
+            context.getRequestManager().sendRequestAsync(updateNetworkConfigRequest, request -> portService.updatePortStatus(request, networkConfig, null));
```

Comment:
So, update doesn't require multiple resource types, then. A comment explicitley stating this fact would be helpful.